src/platform/web/ui/session/room/timeline/BaseMessageView.js

```diff
@@ -24,12 +24,13 @@ import {Menu} from "../../../general/Menu.js";
 import {ReactionsView} from "./ReactionsView.js";
 
 export class BaseMessageView extends TemplateView {
-    constructor(value, interactive = true, tagName = "li") {
+    constructor(value, renderFlags, tagName = "li") {
```

Comment:
when converting to typescript we could use a bitflag enum here, but this is fine for now.